dask/Dockerfile

```diff
@@ -31,7 +31,7 @@ RUN cat /dask.yml \
     | sed -r "s/pyarrow=/pyarrow>=/g" \
     | sed -r "s/pandas=/pandas>=/g" \
     | sed -r "s/numpy=/numpy>=/g" \
-    > dask_unpinned.yml
+    > /dask_unpinned.yml
```

Comment:
Right below, this is referenced as `/dask_unpinned.yml` (absolute path).

That means this has only been working because the working directory happens to be `/`. Making this an absolute path just makes this a little safer, in case that working directory is changed in the base image in the future.